tasks/internal_reconcile_report_orphan/src/use_cases/get_orphans_page.py

```diff
@@ -7,7 +7,7 @@
 def task(
         orphan_record_filter: OrphanRecordFilter,
         orphans_page_storage: OrphansPageStorageInterface,
-        logger: logging.Logger
+        logger: Logger()
```

Comment:
Same question on Logger typehint.

Reply:
to initiate the logger. Doing this here directly.
logger = Logger()